<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SheetEditor</class>
 <widget class="QDialog" name="SheetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sheet Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="toolbox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>70</height>
      </size>
     </property>
     <widget class="QPushButton" name="justify">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align justify</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_justify_64.png</normaloff>image/icon/text_justify_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addCol">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Insert column</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/column_insert_64.png</normaloff>image/icon/column_insert_64.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="italic">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Italic</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_italic_64.png</normaloff>image/icon/text_italic_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addRow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Insert row</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/row_insert_64.png</normaloff>image/icon/row_insert_64.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="bottom">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align bottom</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_align_bottom_64.png</normaloff>image/icon/text_align_bottom_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="textBgColor">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Text background color</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/color_text_background_64.png</normaloff>image/icon/color_text_background_64.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="delRow">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Delete row</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/row_delete_64.png</normaloff>image/icon/row_delete_64.png</iconset>
      </property>
     </widget>
     <widget class="QFontComboBox" name="family">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="lineSpaceIcon">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_line_space_64.png</normaloff>image/icon/text_line_space_64.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="middle">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align middle</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_align_middle.png</normaloff>image/icon/text_align_middle.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="right">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align right</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_right_64.png</normaloff>image/icon/text_right_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="left">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align left</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_left_64.png</normaloff>image/icon/text_left_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bold">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Bold</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_bold_64.png</normaloff>image/icon/text_bold_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="center">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align center</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_center_64.png</normaloff>image/icon/text_center_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="letterSpace">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Letter space</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="textColor">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Text foreground color</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/color_text_64.png</normaloff>image/icon/color_text_64.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="delCol">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Delete column</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/column_delete_64.png</normaloff>image/icon/column_delete_64.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="letterSpaceIcon">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_letter_space_64.png</normaloff>image/icon/text_letter_space_64.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="top">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Align top</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_align_top_64.png</normaloff>image/icon/text_align_top_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="point">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>51</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Font size</string>
      </property>
     </widget>
     <widget class="QPushButton" name="underLine">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Underline</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_underline_64.png</normaloff>image/icon/text_underline_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="wrap">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Wrap</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/text_wrap_64.png</normaloff>image/icon/text_wrap_64.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bgColor">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Background color</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icon/color_background_64.png</normaloff>image/icon/color_background_64.png</iconset>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="lineSpace">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Line space</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
